### ISSUE TITLE
Fix usage sections in Rd files side

### ISSUE THREAD

**Issue by eitsupi** [pola-rs/r-polars#698]
Related to this, we shoud update usage section like the followings:
https://github.com/pola-rs/r-polars/blob/396caf97ec9c21a639851bc0d357823c28771d40/docs/make-docs.R#L67-L69

Maybe something with this:

```r
content_rd <- tools::parse_Rd("man/DataFrame_clone.Rd")
tags <- tools:::RdTags(content_rd)
content_rd[[which(tags == "\\usage")]][[2]][1] <- content_rd[[which(tags == "\\usage")]][[2]][1] |>
  stringr::str_replace("^DataFrame_", "<DataFrame>$")
content_rd |>
  as.character(deparse = TRUE) |>
  readr::write_lines("man/DataFrame_clone.Rd", sep = "")
```

_Originally posted by @eitsupi in https://github.com/pola-rs/r-polars/issues/560#issuecomment-1837412864_

**Comment by etiennebacher** [comment]
I'm really wondering how we can make this work. The code you provide would have to run after each `devtools::document()` call, even when the package is being built by R CMD. It would be nice to have an option to run custom script at the end of `roxygenize()`.

Supposing this is possible, there's probably some code in `altdoc_postprocessing.R` that could be removed.

**Comment by eitsupi** [comment]
> I'm really wondering how we can make this work. The code you provide would have to run after each `devtools::document()` call, even when the package is being built by R CMD.

Wouldn't it be possible to use the same strategy as the script that is currently auto-formatting?

**Comment by eitsupi** [comment]
I implemented this in #701, but it failed due to checking of the usage section by R CMD check.
Perhaps this cannot be achieved.
